packages/angularjs-auth/lib/login/login-controller.js

```diff
@@ -0,0 +1,56 @@
+var CONSTANTS = require('../constants');
+
+/**
+ *
+ * Controller for user login / logout
+ *
+ * @param $timeout
+ *
+ */
+function LoginCtrl($timeout, $http, $state, authService) {
+  var self = this;
+  self.loginErrorMessage = "";
+  self.loginMessages = { success: false, error: false };
+  $timeout(function() {
+    self.hasSession = false;
+  });
+  self.login = function(valid) {
+    if (valid) {
+      self.loginErrorMessage = "";
+      self.loginMessages.error = false;
+
+      $http.post('http://localhost:8001/token', {
+        username: self.username,
+        password: self.password
+      }).then(function(res) {
+        localStorage.setItem(CONSTANTS.TOKEN_CACHE_KEY, res.data.token);
+        localStorage.setItem('rcuser_profile', JSON.stringify(res.data.profile));
+        $http({
```

Comment:
Is that required for login or it's only to check if user is logged in? 
Really strange that we do 2 request on login but I might obviously do not understand intentions here.

Reply:
I was supposed to delete that request but forgot to. It should be removed once I've cleaned up this PR